client/src/Page/Albums.elm

```diff
@@ -54,18 +51,16 @@ type alias Album =
     , name : String
     , imageId : Maybe String
     , artistName : String
-    , createdAt : Date
+    , createdAt : String
     , discs : List Disc
     }
 
 
 dateField name attrs =
-    GraphQL.assume
```

Comment:
Restore `assume`.

(This is likely the cause of the "white page" errors we occasionally see, as this PR is intended to just upgrade Elm any improvements should be left out.)